examples/data_labeler/data_labeler_predict_train.ipynb

```diff
@@ -0,0 +1,370 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "framed-portfolio",
+   "metadata": {},
+   "source": [
+    "# Complex Data Type Detection using Data Labeler component of Data Profiler"
```

Comment:
Just some grammatical / capitalization fixes:
Complex data type detection using the Data Labeler component of the Data Profiler

Reply:
changed